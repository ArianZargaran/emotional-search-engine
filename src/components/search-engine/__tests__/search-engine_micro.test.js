import React from "react";
import SearchEngineMicro from "../search-engine_micro";

it('renders SearchEngineMicro correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<SearchEngineMicro />);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});