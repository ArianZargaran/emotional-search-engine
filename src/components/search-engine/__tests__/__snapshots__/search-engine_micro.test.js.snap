// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SearchEngineMicro correctly 1`] = `
<a
  className="ese-search-engine_micro"
  href="##"
>
  <div
    className="ese-search-engine_micro_msg"
  >
    <div
      className="ese-search-engine_micro_arrow"
    />
    <p
      className="ese-search-engine_micro_txt"
    >
      Search 
      <br />
      by voice
    </p>
  </div>
</a>
`;
